Themis/Views/Assessment/CorrectionSidebar/EditFeedback/EditFeedbackViewBase.swift

```diff
@@ -39,7 +39,9 @@ struct EditFeedbackViewBase: View {
     func createFeedback() {
         feedback.detailText = detailText
         feedback.credits = score
-        cvm.addInlineHighlight(feedbackId: feedback.id)
+        if type == .inline {
+            cvm.addInlineHighlight(feedbackId: feedback.id)
+        }
```

Comment:
I don't exactly know why the `if` isn't there on the develop branch, but it should be correct I believe.

Reply:
The if is required for a bugfix, made a comment on that in the previous PR.